app/decorators/mission_decorator.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+# MissionDecorator
+class MissionDecorator < Draper::Decorator
+  delegate_all
+
+  def search_slots_selectable_by_current_member
+    start_times = object.slots.group(:start_time).count.keys
+    selectable_slots = []
+
+    start_times.each do |start_time|
+      slot = mission.slots.where(start_time: start_time, member_id: h.current_member.id).first
+      slot = mission.slots.where(start_time: start_time).first  if slot.nil?
```

Comment:
is `mission` here actually `object` ?

Reply:
is a useless method that forgot to remove

Reply:
solved